gbpservice/contrib/nfp/tools/image_builder/disk_image_create.py

```diff
@@ -35,7 +36,7 @@ def create_configurator_docker():
     docker_images = os.path.realpath(docker_images)
 
     # create a docker image
-    os.chdir(cur_dir)
+    os.chdir(docker_build_dir)
```

Comment:
This makes us to invoke the disk image script from image_builder directory only right ?